dbt_project.yml

```diff
@@ -1360,4 +1360,17 @@ models:
     openxswap:
       +schema: openxswap
       optimism:
-        +schema: openxswap_optimism
\ No newline at end of file
+        +schema: openxswap_optimism
+
+    boost:
+      +schema: boost
+      optimism:
+        +schema: boost_optimism
+      ethereum:
+        +schema: boost_ethereum
+      arbitrum:
+        +schema: boost_arbitrum
+      base:
+        +schema: boost_base
+      polygon:
+        +schema: boost_polygon
```

Comment:
additions to the project file are no longer needed -- we're looking to maintain schema names directly in model config blocks